Integrations/Cymulate/Cymulate.yml

```diff
@@ -0,0 +1,67 @@
+category: Vulnerability Management
+commonfields:
+  id: Cymulate
+  version: -1
+configuration:
+- defaultvalue: 179743a665726a99fdee3bb0d10ba3fe
+  display: x-token
+  name: x-token
+  required: true
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
```

Comment:
add 2 parameters that we add to all the integrations.
```
- display: Trust any certificate (not secure)
  name: insecure
  required: false
  type: 8
- display: Use system proxy settings
  name: proxy
  required: false
  type: 8
```

Reply:
use both of them in the code.
insecure should be be passed to `_http_request` to `verify` argument. It verifies the server certificate (allowing the user to choose the secure version)